.github/workflows/publish-abac-nuget.yml

```diff
@@ -0,0 +1,36 @@
+name: Pack and publish Altinn.Authorization.ABAC nuget package
+
+on:
+  pull_request:
+    branches: infrastructure/publish-nuget-action
+  release:
+    types:
+      - published
+
+jobs:
+  build-pack:
+    #if: startsWith(github.ref, 'refs/tags/Altinn.Authorization.ABAC-')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: |
+            2.0.x
+      - name: Install deps
+        run: |
+          cd src/Altinn.Authorization.ABAC
+          dotnet restore
+      - name: Build
+        run: |
+          cd src/Altinn.Authorization.ABAC
+          dotnet build --configuration Release --no-restore -p:Deterministic=true -p:BuildNumber=${{ github.run_number }}
+      - name: Pack and publish Altinn.Authorization.ABAC
```

Comment:
I would split pack and publish into two steps. In general, having smaller steps makes it easier to figure out what went wrong when something doesn't work.